CONTRIBUTING.md

```diff
@@ -0,0 +1,158 @@
+evaluation, training, and deployment of GenAI, Deep Learning, and Traditional ML models.
+Sharing your use cases, tips, and best practices leveraging MLflow is a great way to contribute to the community, enhance your writing
+portfolio, and help other MLflow users by sharing your valuable knowledge and experience.
+
+## Content That We're Looking For
+
+We welcome submissions on a wide range of topics, including but not limited to:
+
+* How-tos, technical guides and tutorials relating to the use of MLflow for GenAI, DL, or ML use cases
+* Deep dives into specific features within MLflow
+* Use Case studies relating to the use of MLflow
+* Best practices of using complex or advanced features within MLflow
+* Tips and tricks for leveraging MLflow for common tasks in the ML Development lifecycle
+* Overview and introduction to major new to-be released features in MLflow
+* Summaries and reports from MLflow meetups or summit events
+
+> Note: If you suggest a topic not listed above, we'll be happy to discuss its applicability to the blog within your filed proposal.
+
+## Submission Process
+
+* **Idea Proposal**: Before writing, submit a brief outline of your idea via an MLflow Website Issue submission using the ["Blog Post"](https://github.com/mlflow/mlflow-website/issues) issue type. Ensure that all elements are filled in within the filed issue, paying careful attention to the legal acknowledgement to ensure that you don't spend time writing about something that will never be able to be published.
+
+* **Discussion and Revision**: An MLflow maintainer or Developer Relations member may request further clarification to the content of your blog from within the filed issue or suggestions and direction on the overall content of the blog.
+
+* **Approval**: Wait for approval from one of the MLflow maintainers prior to working on your blog (to save you time and frustration). Maintainers will have an open dialog with you regarding the scope, content, and chosen topics to ensure that the blog content aligns with our vision of enabling open source ML developers to build effective ML projects.
+
+* **PR Submission**: Once the subject of your blog has been approved (with confirmation from an MLflow maintainer), write your blog post following the guidelines provided below. File a PR from your feature branch and select review from the maintainer that approved your proposal from the filed Issue.
+
+* **Review Process**: An MLflow maintainer will review your PR and provide feedback, corrections, and general guidance on adjusting the content, pacing, writing style, and embedded content to ensure that the PR meets with the standards for the MLflow website. As corrections are made and changes are pushed, work with the reviewer to get your blog PR in a state that it can be merged to the main branch.
+
+* **Publication**: After your PR is merged, your blog post will be released with the next minor release of MLflow.
```

Comment:
I think we can release a new blog anytime we want. Can we adjust the wording here?

Reply:
We don't want to set that precedent :) Otherwise, you and I will be pinged daily about "could you release my blog?". I want external blogs to match with our release cadence to limit the amount of questions and pings that we get. We're going to be having ~20-30 blogs / quarter and I don't want us to do more than 2-3 releases per quarter :)

Reply:
Can we add a link the the Mlflow Project here.. So people know what the next releases are..

I agree with @harupy we will probably need a process in the future that allow folks to publish after it is approved, rather than waiting for the MLflow version release

Reply:
They won't be releasing. We would have to release. And we're not taking on the maintenance burden of releasing 5 blogs / week.